src/torch/gradchecks.py

```diff
@@ -0,0 +1,156 @@
+import os
```

Comment:
this file should be converted to a pytest. Maybe @paskino can help.

Reply:
@paskino I tried to turn this into a pytest